public/instances.json

```diff
@@ -1,4 +1,33 @@
 {
+  "P2B Compliance": {
+    "languages": ["*eu"],
+    "jurisdictions": ["EU"],
+    "stats": {
+      "services": "208",
+      "documents": "253"
+    },
+    "industries": {
+      "en": "Online intermediation services for businesses subject to the European <a href=\"https://eur-lex.europa.eu/eli/reg/2019/1150/oj\">platforms-to-businesses (“P2B” / 2019/1150) regulation</a>",
+      "fr": "Services d’intermédiation en ligne pour les entreprises sujets au règlement européen <a href=\"https://eur-lex.europa.eu/legal-content/FR/TXT/HTML/?uri=CELEX:32019R1150&from=EN\">P2B / 2019/1150</a>"
+    },
+    "maintainers": [
+      {
+        "name": "Ambassadeur français pour le numérique",
+        "logo": "/images/contributors/ambnum.png",
+        "url": "https://disinfo.quaidorsay.fr/fr"
+      },
```

Comment:
I don't think so 🤔 
```suggestion
```

Reply:
I'm not sure I understand, what do you mean ?

Reply:
OK, I've updated the list of maintainers to reflect this.